backend/blueprint/api/helpers/uploads.py

```diff
@@ -9,7 +9,8 @@ def generateFileName():
 
 def saveToImage(imageFile=None, extension='.png'):
     imageName = generateFileName() + extension
-    imageDirectory = os.path.join(app.config.get('BASE_DIR'), 'static', 'uploads', 'image')
+    imageDirectory = os.path.join(app.config.get(
```

Comment:
No need of this as the previous length was fine according to hound configurations :+1: